modin/pandas/series.py

```diff
@@ -15,7 +15,162 @@ def na_op():
+
+    def __mod__(self, other):
+        return self.__arithmetic_op__("__mod__")(other)
+
+    def __pow__(self, other):
+        return self.__arithmetic_op__("__pow__")(other)
+
+    def __radd__(self, other):
+        return self.__arithmetic_op__("__radd__")(other)
+
+    def __rmul__(self, other):
+        return self.__arithmetic_op__("__rmul__")(other)
+
+    def __rsub__(self, other):
+        return self.__arithmetic_op__("__rsub__")(other)
+
+    def __rtruediv__(self, other):
+        return self.__arithmetic_op__("__rtruediv__")(other)
+
+    def __rfloordiv__(self, other):
+        return self.__arithmetic_op__("__rfloordiv__")(other)
+
+    def __rmod__(self, other):
+        return self.__arithmetic_op__("__rmod__")(other)
+
+    def __rpow__(self, other):
+        return self.__arithmetic_op__("__rpow__")(other)
+
+    def __iadd__(self, other):
+        return self.__arithmetic_op__("__iadd__")(other)
+
+    def __imul__(self, other):
+        return self.__arithmetic_op__("__imul__")(other)
+
+    def __isub__(self, other):
+        return self.__arithmetic_op__("__isub__")(other)
+
+    def __itruediv__(self, other):
+        return self.__arithmetic_op__("__itruediv__")(other)
+
+    def __ifloordiv__(self, other):
+        return self.__arithmetic_op__("__ifloordiv__")(other)
+
+    def __imod__(self, other):
+        return self.__arithmetic_op__("__imod__")(other)
+
+    def __ipow__(self, other):
+        return self.__arithmetic_op__("__ipow__")(other)
+
+    def __neg__(self, other):
+        return self.__arithmetic_op__("__neg__")(other)
+
+    def __iter__(self):
+        return self.series.__iter__()
+
+    def __len__(self):
+        return self.series.__len__()
+
+    def __getattribute__(self, item):
+        default_behaviors = [
+            "__init__",
+            "series",
+            "parent_df",
+            "_loc",
+            "__arithmetic_op__",
+            "__comparisons__",
+        ]
+        if item not in default_behaviors:
+            method = self.series.__getattribute__(item)
+            if (
+                callable(method)
+                and "inplace" in str(inspect.signature(method))
+                and self.parent_df is not None
+            ):
+
+                def inplace_handler(*args, **kwargs):
+                    if kwargs.get("inplace", False):
+                        prev_len = len(self.series)
+                        self.series.__getattribute__(item)(*args, **kwargs)
+                        if prev_len == len(self.series):
+                            self.parent_df[self._loc] = self.series
+                        else:
+                            self.parent_df.reindex(index=self.series.index, copy=False)
+                        return None
+                    else:
+                        return self.series.__getattribute__(item)(*args, **kwargs)
+
+                method = inplace_handler
+            return method
+        elif item not in default_behaviors:
```

Comment:
```suggestion
         elif item in default_behaviors:
```
IDK if this is the intention, but we have two repeated `item not in default_behaviors`

Reply:
Good catch, the second conditional was created during development for testing and should be removed.